docker/dev/build.sh

```diff
@@ -88,6 +84,14 @@ git fetch origin # The residual local repo might not be on a branch.
 git checkout 36a2f217f3311d861cf736894ecb5147b6e44c74
 python setup.py bdist_wheel -q --dist-dir $SQLFLOW_BIN > /dev/null
 
+echo "Build couler-proj/couler ..."
+cd $SQLFLOW_BIN
+if [[ ! -d couler ]]; then
+    git clone https://github.com/couler-proj/couler.git
+fi
+git checkout 374716477964ef68284addabe9dcd322f3a18c3b 
```

Comment:
Missing `cd couler` here

Reply:
The latest commit is now `c52d0841de0a4b1b574f2c38385dfdf57e9f3ebd` if you'd like to incorporate some of the recent fixes.